doc/classes/OS.xml

```diff
@@ -54,7 +54,9 @@
 			<argument index="0" name="msec" type="int">
 			</argument>
 			<description>
-				Delay execution of the current thread by [code]msec[/code] milliseconds. [code]usec[/code] must be greater than or equal to [code]0[/code]. Otherwise, [method delay_msec] will do nothing and will print an error message.
+				Delays execution of the current thread by [code]msec[/code] milliseconds. [code]usec[/code] must be greater than or equal to [code]0[/code]. Otherwise, [method delay_msec] will do nothing and will print an error message.
+				[b]Note:[/b] [method delay_msec] is a [i]blocking[/i] way to delay code execution. To delay code execution in a non-blocking way, use [code]yield(get_tree().create_timer(TIME), "timeout")[/code] where [code]TIME[/code] is the duration to wait for in [i]seconds[/i]. This will delay the execution of code placed below the [code]yield[/code] without affecting the rest of the project (or editor, for [EditorPlugin]s and [EditorScript]s).
```

Comment:
It may be better to link to the documentation for `SceneTree.create_timer()`, so we aren't duplicating its description here.

```suggestion
				[b]Note:[/b] [method delay_msec] is a [i]blocking[/i] way to delay code execution. To delay code execution in a non-blocking way, see [method SceneTree.create_timer].
```

Alternatively:

```suggestion
				[b]Note:[/b] [method delay_msec] is a [i]blocking[/i] way to delay code execution. To delay code execution in a non-blocking way, use [code]yield(get_tree().create_timer(TIME), "timeout")[/code] where [code]TIME[/code] is a [float] representing the duration to wait for in [i]seconds[/i]. This will delay the execution of code placed below the [code]yield[/code] without affecting the rest of the project (or editor, for [EditorPlugin]s and [EditorScript]s).
```